gradle.properties

```diff
@@ -0,0 +1,14 @@
+useLocal=false
+pluginSinceBuild=241.14494.240
+pluginUntilBuild=242.*
+
+javaVersion=17
+javaTargetVersion=17
+
+# Target IntelliJ Community by default
+platformType=IC
+platformVersion=2024.1.7
+
+# Example: platformBundledPlugins = com.intellij.java
+platformBundledPlugins=com.intellij.java, org.jetbrains.idea.maven, com.intellij.gradle, org.jetbrains.plugins.terminal, com.intellij.properties
```

Comment:
Comparing this list of plugins to the original list... https://github.com/OpenLiberty/liberty-tools-intellij/blob/3e3c97c89d475acbf9888f781a17cb1bd97cfd87/build.gradle#L151... I do not see "maven" or "gradle-java"

Reply:
According to the new IntelliJ Plugin documentation( https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html#bundled-and-other-plugins), it is recommended to use plugin IDs rather than plugin names when defining dependencies in your plugin's gradle.properties file 
Maven Plugin:
The plugin ID for the Maven integration in IntelliJ IDEA is indeed org.jetbrains.idea.maven. This one is already given 
Gradle Plugin:
There is no such plugin ID for Gradle-java, this can be considered as 2 different plugin i.e
Java plugin: The plugin ID for Java support is com.intellij.java.
Gradle plugin: The plugin ID for Gradle support is com.intellij.gradle.